### ISSUE TITLE
Module not found error with tailwindcss

### ISSUE THREAD

**Issue by jagoodhand** [reaviz/reaviz#216]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

With the latest version of reaviz, I get the error below when trying to either build or start the dev server using create react app:

```bash
Module not found: Error: Can't resolve 'tailwindcss/plugin' in '/PATH_TO_MY_PROJECT/node_modules/tw-colors/dist/lib'
Did you mean 'plugin.js'?
BREAKING CHANGE: The request 'tailwindcss/plugin' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
WARNING in ./node_modules/@reaviz/react-use-fuzzy/lib/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/PATH_TO_MY_PROJECT/node_modules/@reaviz/react-use-fuzzy/src/index.ts' file: Error: ENOENT: no such file or directory, open '/PATH_TO_MY_PROJECT/node_modules/@reaviz/react-use-fuzzy/src/index.ts'

ERROR in ./node_modules/tw-colors/dist/lib/index.mjs 3:0-40
Module not found: Error: Can't resolve 'tailwindcss/plugin' in '/PATH_TO_MY_PROJECT/node_modules/tw-colors/dist/lib'
Did you mean 'plugin.js'?
BREAKING CHANGE: The request 'tailwindcss/plugin' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

## Expected behavior

Project builds correctly.

I do not get this issue when I roll back to reaviz 14.15.4

## Minimal reproduction of the problem with instructions

* yarn start

## Environment

<pre><code>
Libs:
- react version: 18.2.0
- reaviz version: 15.15.0

Browser:
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 20.9.0
- Platform:  Mac

**Comment by ghsteff** [comment]
thanks for reporting - do you mind providing a github repo reproducing the issue please?

**Comment by ghsteff** [comment]
This should be fixed in 15.15.1

**Comment by jagoodhand** [comment]
Yes that seems to have fixed it for me. Thanks! Sorry I didn't get a chance to put together the repo you requested.
